modules/tinymce/src/plugins/table/main/ts/ui/DomModifier.ts

```diff
@@ -8,29 +8,23 @@ export interface DomModifier {
 
 // The get node is required here because it can be transformed
 // when switching between tags (e.g. th and td)
-const modifiers = (testTruthy: boolean) => (editor: Editor, element: Element): DomModifier => {
+const modifiers = (editor: Editor, element: Element): DomModifier => {
```

Comment:
This should probably just be called `normal` now given it's not used to generate different DOM modifiers anymore.